widget_driver/test/test_helpers/test_container_drivable_widget.dart

```diff
@@ -0,0 +1,60 @@
+import 'package:flutter/material.dart';
+import 'package:widget_driver/widget_driver.dart';
+
+class TestContainerDriver extends WidgetDriver {
```

Comment:
I think some docu for the classes in this file would help if someone wants to add tests in the future.

Reply:
Sounds good. I will add some.

Reply:
Okay, I updated the PR now.